content/docs/guides/testing/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Testing
+meta_desc: "Guide to automated testing of Pulumi programs: unit testing, integration testing, Policy as Code."
+menu:
+    userguides:
+        identifier: testing
+        weight: 7
+---
+
+Pulumi uses general-purpose programming languages to define cloud infrastructure. This capability opens up all the doors to **automated testing** of the infrastructure resources.
+
+The software industry has come up with multiple styles of automated testing. Unit testing of individual pieces, integration testing of system components, end-to-end testing of the complete application&mdash;each style has its advantages and challenges.
+
+Pulumi provides a similar choice of testing options for cloud programs. Currently, we identify three types of tests:
+
+- **Unit Tests** are fast in-memory tests that mock all external calls.
+- **Tests based on Policy as Code** run resource-level assertions *while* infrastructure is being deployed.
+- **Integration Tests** deploy ephemeral infrastructure and run external tests against it.
```

Comment:
Should these bullets be in the same order as the columns in the table and headers below?

Reply:
Makes sense 👍

Reply:
Re-ordered